src/scss/main.scss

```diff
@@ -11,27 +13,51 @@ margin:0px;
 }
 
 .nav ul{
+  @include media($mobile){
+    @include span-columns(12);
+    text-align:center;
+    padding-left:0px;
+  }
+  @include media ($desktop){
+    @include span-columns(5);
+    float:right;
+  }
+  @include media ($tablet){
+    @include span-columns(12);
+    text-align:center;
```

Comment:
Put spaces after the `:` in your rules. A good guide for general code style is https://github.com/airbnb/css, these are super valuable to get into.